Documentation/production-ready.md

```diff
@@ -1,4 +1,6 @@
-etcd is being used successfully by many companies in production. It is,
-however, under active development and systems like etcd are difficult to get
-correct. If you are comfortable with bleeding-edge software please use etcd and
+# Etcd in Production
```

Comment:
we try to avoid doing Etcd, and the general style is etcd. Makes starting a sentence with etcd hard though. Same for rkt.